sunpy/coordinates/ephemeris.py

```diff
@@ -14,6 +14,17 @@
 from astropy.coordinates.representation import CartesianRepresentation, SphericalRepresentation
 from astropy._erfa.core import ErfaWarning
 from astropy.constants import c as speed_of_light
+# Versions of Astropy that do not have HeliocentricMeanEcliptic have the same frame
+# with the incorrect name HeliocentricTrueEcliptic
+try:
+    from astropy.coordinates import HeliocentricMeanEcliptic
+except ImportError:
+    from astropy.coordinates import HeliocentricTrueEcliptic as HeliocentricMeanEcliptic
+try:
+    from astroquery.jplhorizons import Horizons
```

Comment:
We will want to add this as an extra dependency. I am not sure what group.

Reply:
I am really not sure we do, astroquery is continuously deployed because it's always adapting to upstream changes. It might be best to just have users install this if they need this function?

Reply:
Well if that's the case, I think we should move this import into the function and make a note that it is required top be installed separately.

Reply:
I agree with that.